lib/chef/resource/rhsm_register.rb

```diff
@@ -69,11 +70,13 @@ class RhsmRegister < Chef::Resource
           remote_file "#{Chef::Config[:file_cache_path]}/katello-package.rpm" do
             source "http://#{new_resource.satellite_host}/pub/katello-ca-consumer-latest.noarch.rpm"
             action :create
-            notifies :install, "yum_package[katello-ca-consumer-latest]", :immediately
+            notifies :install, "{rhel8? ? 'dnf' : 'yum'}_package[katello-ca-consumer-latest]", :immediately
```

Comment:
should this be a helper? We do it a lot...

Also, I worry about rhel9 breaking. I feel like this should be something like `(rhel? && platform_version.to_i < 8) ? 'yum' : 'dnf'`

Reply:
That's fair. Let me rework that.